src/sdk/config_fs.js

```diff
@@ -468,6 +496,46 @@ class ConfigFS {
         return nb_native().fs.stat(this.fs_context, path_for_account_or_user_config_file);
     }
 
+    /**
+     * stat_account_config_file_by_identity will return the stat output on account config file by id or by account name
+     * @param {string} id
+     * @param {string} account_name
+     * @returns {Promise<nb.NativeFSStats>}
+     */
+    async stat_account_config_file_by_identity(id, account_name) {
+        const identity_path = this.get_identity_path_by_id(id);
+        let stat;
+        try {
+            const stat_by_identity_path = await nb_native().fs.stat(this.fs_context, identity_path);
+            stat = stat_by_identity_path;
+        } catch (err_stat_by_identity_path) {
+            if (err_stat_by_identity_path.code === 'ENOENT') {
```

Comment:
this nested levels of try-catch are a bit confusing. maybe we can move those condition outside the try-catch, and return when a condition is met. what do you think?

Reply:
I didn't understand it, could you give an example or add your suggestion?

Reply:
maybe you can do something like:
```
try {
    stat = await nb_native().fs.stat(this.fs_context, identity_path);
    return stat;
catch(err) { 
    if (err_stat_by_identity_path.code !== 'ENOENT') {
        throw err
    }
}
try {
     stat = await nb_native().fs.stat(this.fs_context, account_name_new_path);
    return stat;
} catch (err_stat_by_account_name_new_path
...
```
something like this
that way the conditions are separated from each other.

Reply:
I changed it according to your suggestion, thanks.